src/Makefile.qt.include

```diff
@@ -394,8 +394,7 @@ bitcoin_qt_apk: FORCE
 	cp $(dir $(lastword $(CC)))../sysroot/usr/lib/$(host_alias)/libc++_shared.so $(APK_LIB_DIR)
 	tar xf $(QT_BASE_PATH) -C qt/android/src/ $(QT_BASE_TLD)src/android/jar/src --strip-components=5
 	tar xf $(QT_BASE_PATH) -C qt/android/src/ $(QT_BASE_TLD)src/android/java/src --strip-components=5
-	tar xf $(QT_BASE_PATH) -C qt/android/res/ $(QT_BASE_TLD)src/android/java/res --strip-components=5
```

Comment:
d09d14c1d31e14b69abcf0d030e87b475bd084a0

Why is this line removed?

Reply:
Because I found out that the only thing we are extracting from there are the strings (in many languages) to do with Ministro (which is a way of fetching Qt libs at runtime). This wasn't used and isn't likely to be in the furure. That whole directory used to be .gitignored but now we have `values/libs.xml` there so two birds one stone

Reply:
src/Makefile.qttest.include